content/applications/sales/subscriptions/renewals.rst

```diff
@@ -1,80 +1,70 @@
-*Renewal quotation*. By clicking on it, a new quotation is created. From there, start a basic sales
-flow allowing you to send the quotation by email to your customers or to confirm it. It is better to
-first *Send by email* the quotation to your customers in order to have their confirmation and, then,
-*Confirm* it in **Odoo Sales**.
+When the :guilabel:`Renew` is clicked, Odoo instantly presents a new renewal quotation, complete
+with a :guilabel:`Renewal Quotation` tag.
 
-.. note::
-   In the Chatter of this new quotation, it is mentioned that "This renewal order has been created
-   from the previous subscription". Once confirmed by your customers, this quotation becomes a
-   sales order and a new sale is mentioned in the upper right corner of the subscription.
+.. image:: renewals/renewal-quotation.png
+  :align: center
+  :alt: Renewal quotation in the Odoo Subscriptions application.
 
-   .. image:: renewals/renew-a-quotation.png
-     :align: center
-     :alt: Renew a quotation with Odoo Subscriptions
+From here, a standard sales flow can occur, which is typically started by clicking :guilabel:`Send
+by Email`. This sends a copy of the quotation to the customer for them to confirm, and eventually,
+pay for.
 
-   By clicking on the *Sales* button, you have a summary of your sales orders in a list view.
-   The only difference between your two quotations is the description underneath the
-   *Subscription Management* category. There, you can easily visualize which one is your renewal.
+.. note::
+  In the *Chatter* of the :guilabel:`Renewal Quotation`, it is mentioned that this subscription is
+  the renewal of the subscription from the original sales order.
 
-   .. image:: renewals/subscription-management-category.png
-     :align: center
-     :alt: Renewal as Subscription Management form in Odoo Subscriptions
+Once the :guilabel:`Renewal Quotation` is confirmed, it becomes a sales order, and a
+:guilabel:`Sales History` smart button appears at the top of the page.
 
-Visualize your subscriptions to renew
-=====================================
+.. image:: renewals/sales-history-smart-button.png
+  :align: center
+  :alt: Sales History smart button in the Odoo Subscriptions application.
 
-Finally, if you want to visualize all your subscriptions and easily find those to renew, you can go
-to your *Subscriptions dashboard* and use the filter *To renew*.
+When that :guilabel:`Sales History` smart button is clicked, Odoo reveals a separate page,
+showcasing the different sales orders attached to this subscription, along with their individual
+:guilabel:`Subscription Status`.
 
-.. image:: renewals/subscriptions-dashboard-with-the-to-renew-filter.png
+.. image:: renewals/sales-history-page.png
   :align: center
-  :alt: List view of all subscriptions and use of the filter to renew in Odoo Subscriptions
+  :alt: Renewal quotation in the Odoo Subscriptions application.
```

Comment:
I believe once a renewal is confirmed the **MRR** smartbutton becomes available too. Can we talk about this?